### DISCUSSION TITLE
Question on binding Gizmos to FMS for practice day and then rebinding them to driver's station for final week before checking in for the competition

### DISCUSSION THREAD

**Discussion by shannonsjones** [gizmo-platform/gizmo#26] [Q&A]
I was hoping to run the FMS during our practice / mall day event, which I presume would mean that we need to bind all of the Gizmos to the FMS at the practice day.  I believe that means that they would no longer work in wireless mode after the practice day, unless they rebind the gizmo to their driver's station so that it can work wirelessly for the time after the practice day event and before the competition event.  

Will binding the Gizmos to the FMS for practice day, then rebinding them to the driver's station afterwards, then rebinding them to the FMS on competition check in cause any issues, or is that going to be an acceptable sequence of events?

I realize once bound to the FMS for practice day, the teams will need to be tethered in their pit spaces during the practice day, until the are re-bound to their driver's station.

**Comment by the-maldridge** [comment]
I thought for sure I'd sent a reply to this yesterday but didn't actually click the button...

Your  understanding is correct of the binding sequence, and it will work as expected.  There are some security implications to re-using network keys between practice and competition days, though I would say they're probably acceptable risks at this time.  I can give you more accurate information on that not on a public thread.

**Comment by shannonsjones** [comment]
Thanks.  If we do end up using the FMS for practice day, I will either tell
teams to rebind them to their drivers stations or have them come see me and
I will do it before they leave.
